lib/output.js

```diff
@@ -67,6 +72,20 @@ module.exports = function (parameters, response) {
     threshold
   ));
 
+  if (parameters.optimized) {
+    console.log('\nHere are your optimized images: ', humanizeUrl(optimizedResoruceURL));
+  }
+
+  if (parameters.download) {
+      new Download({mode: '755'})
```

Comment:
Why did you switch out `got`? I see no reason to use `download` here as it has a very heavy dependency tree and `got` does exactly what we need.